.travis.yml

```diff
@@ -21,16 +19,20 @@ addons:
       - llvm-4.0-dev
       - clang-4.0
       - lib32z1-dev
+      - python2.7
+      - python3.6
 
 install:
-  - sudo rm -f `which llvm-config`
-  - export WELD_HOME=`pwd`
+  - pip install virtualenv==15.1.0
+  - mkdir -p .virtualenv
+  - export VENV_HOME=`pwd`/.virtualenv
+  - virtualenv $VENV_HOME/py2.7
```

Comment:
It might be best to be explicit with both of these, e.g.:

```
- virtualenv $VENV_HOME/python2.7 --python=python2.7
- virtualenv $VENV_HOME/python3.6 --python=python3.6
```

You'll also want to `deactivate` below before `activate`-ing a different virtualenv. You can probably wrap it all in a helper script like:

```shell
 #!/bin/bash

$PYTHON_VERSIONS=$@

pip install virtualenv==15.1.0
mkdir -p .virtualenv
export VENV_HOME=`pwd`/.virtualenv

for v in $PYTHON_VERSIONS; do
  virtualenv "$VENV_HOME/python$v" --python="python$v"
  source "$VENV_HOME/python$v/bin/activate"
  pip install --upgrade pip setuptools wheel
  pip install --only-binary=numpy,pandas -r requirements.txt
  deactivate
done
```

and then call:

```
  - init_virtualenvs.sh 2.7 3.6
```